samples/extensionSamples/yarn.lock

```diff
@@ -2345,13 +2345,6 @@ mimic-response@^3.1.0:
   resolved "https://registry.yarnpkg.com/mimic-response/-/mimic-response-3.1.0.tgz#2d1d59af9c1b129815accc2c46a022a5ce1fa3c9"
   integrity sha512-z0yWI+4FDrrweS8Zmt4Ej5HdJmky15+L2e6Wgn3+iK5fWzb6T3fhNFq2+MeTRb064c6Wr4N/wv0DzQTjNzHNGQ==
 
-"minimatch@2 || 3", minimatch@^3.0.3, minimatch@^3.0.4:
```

Comment:
Huh, so guess these were just not updated after a package.json change?

Reply:
I think there's a problem with yarn, I had to manually delete all of the references to get it to regenerate all of these